webpack.config.prod.js

```diff
@@ -16,11 +16,6 @@ module.exports = webpackMerge.smart(config, {
     },
     devtool: 'source-map',
     plugins: [
-        new webpack.optimize.AggressiveMergingPlugin({
```

Comment:
👍

Reply:
I like this idea, but when I tried it, I think it caused a number of discrete smaller chunks to be created, that probably did feel right to aggressively merge. Do you have more chunks after removing this plugin? I just hate that magic number!